scripts/analysis/amwg_table.py

```diff
@@ -295,17 +302,39 @@ def _spatial_average(indata):
 
 #####
 
-def _write_html(f, out):
+def _write_html(f, out,case_name,case_name_list):
+    if case_name == case_name_list[0]:
+        case = "Test"
+    else:
+        case = "Control"
     import pandas as pd
     df = pd.read_csv(f)
     html = df.to_html(index=False, border=1, justify='center', float_format='{:,.3g}'.format)  # should return string
-    preamble = f"""<html><head></head><body><h1>{f.stem}<h1>"""
+    #preamble = f"""<html><head></head><body><h1>{f.stem}<h1>"""
+    preamble = f"""<html><head><title>ADF Mean Tables</title><link rel="stylesheet" href="../templates/adf_diag.css"></head><body >
+
+    <nav role="navigation" class="primary-navigation">
+      <ul>
+        <li><a href="../index.html">Case Home</a></li>
+        <li><a href="../html_table/mean_table.html">Case Tables</a></li>
+        <li><a href="#">Links &dtrif;</a>
+          <ul class="dropdown">
+            <li><a href="https://www.cesm.ucar.edu">CESM</a></li>
+            <li><a href="https://www.cesm.ucar.edu/working_groups/Atmosphere/?ref=nav">AMWG</a></li>
+            <li><a href="https://www.cgd.ucar.edu/amp/">AMP</a></li>
+          </ul>
+        </li>
+        <li><a href="https://github.com/NCAR/ADF">About</a></li>
+        <li><a href="https://github.com/NCAR/ADF/discussions">Contact</a></li>
+      </ul>
+    </nav><h1>AMP Diagnostics</h1><h2>{case} Case: {f.stem}<h2>"""
```

Comment:
I might change this header to be something like `Model Diagnostics`, `CAM Diagnostics`, or `ADF Diagnostics`.